flipperdevices/Flipper-Android-App
# Move core:ui to multipaltform

**Background**

Some core-ui modules still uses android-only implementations. This PR moves more modules into multiplatform. Except coil and lottie

**Changes**

- Move more core-ui modules into commonMain

**Test plan**

- Open application
- See images and icons are ok
- See dialogs are ok
